felt-screen-api/src/main/java/io/github/feltmc/feltapi/mixin/screen/HandledScreenMixin.java

```diff
@@ -0,0 +1,61 @@
+package io.github.feltmc.feltapi.mixin.screen;
+
+import io.github.feltmc.feltapi.api.screen.HandledScreenExtension;
+import net.minecraft.client.gui.screen.ingame.HandledScreen;
+import net.minecraft.screen.slot.Slot;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+
+@Mixin(HandledScreen.class)
+public class HandledScreenMixin implements HandledScreenExtension {
+
+    @Shadow @Nullable protected Slot focusedSlot;
```

Comment:
why new lines inbetween vars?

Reply:
Oh oops

Reply:
Oh actually blame the minecraft dev plugin